src/doc/trpl/iterators.md

```diff
@@ -242,6 +242,9 @@ for num in nums.iter() {
 }
 ```
 
+Sometimes you need this functionality, but since for loops operate on the
+`IntoIterator` trait, calling `.iter()` is rarely necessary.
```

Comment:
It's not that rare; any time one wishes to call the iterator adaptor methods it is needed.